packages/storybook/src/nimble/table/table.mdx

```diff
@@ -9,9 +10,23 @@ The <Tag name={tableTag}/> is a component that offers a way to render tabular da
 For information about configuring table columns, see **Table Column Configuration** and **Table Column Types**.
 
 <Canvas of={tableStories.table} />
+
+## API
+
 <Controls of={tableStories.table} />
+<ComponentApisLink />
+
+## Styling
+
+### Sizing
+
+The <Tag name={tableTag}/> should be sized explicitly or sized to fill the space of a parent container. The <Tag name={tableTag}/> does not currently support being styled with `height: auto`. The ability to auto size the table is tracked with [issue 1624](https://github.com/ni/nimble/issues/1624).
+
+{/* ## Usage */}
 
-## Delay-loaded Hierarchy
+## Examples
+
+### Delay-loaded Hierarchy
```

Comment:
Should we use sentence case for these headings rather than title case? We used sentence case in the "Table Column Configuration" examples.